contributing/releasing.md

```diff
@@ -84,14 +86,14 @@ Other than the steps above regarding hotfixing, the entire release process stays
 
 ### Start internal testing
 
-You can now start the internal release testing process documented at [go/mdc-releasing](http://go/mdc-releasing).
+You can now start the internal release testing process documented at [go/mdc-releasing](http://go/mdc-releasing#import-the-release-candidate).
 
-### Resolve any failures
+#### Resolve any failures
```

Comment:
Since this starts a new section after using the internal guide, I think it should be either an H2 ("##", my preference) or an H3 ("###", as it was before). Otherwise, it seems like "Resolve any failures" is a sub-step of "Start internal testing". Rather, it appears to be general, ongoing advice for the entire process.